### ISSUE TITLE
bbl to aws us-west-1 fails coz "us-west-1b" for availabilityZone invalid

### ISSUE THREAD

**Issue by dkoper** [cloudfoundry/bosh-bootloader#162]
On `bbl 3.2.6 (linux/amd64)`, stack creation failed:
```
19:01:45 UTC+1000 | CREATE_FAILED | AWS::EC2::Subnet | InternalSubnet2 | Value  (us-west-1b) for parameter availabilityZone is invalid. Subnets can  currently only be created in the following availability zones:  us-west-1a, us-west-1c.
```

Command run:
```
diesk@cloud-cf:~/workspace$ bbl up \
> --aws-access-key-id  my-id \
> --aws-secret-access-key my-key \
> --aws-region us-west-1 \
> --iaas aws
step: checking if keypair "keypair-bbl-env-huron-2017-07-19t09-00z" exists
step: creating keypair
step: generating cloudformation template
step: checking if cloudformation stack "stack-bbl-env-huron-2017-07-19t09-00z" exists
step: creating cloudformation stack
~~~~~

CloudFormation failure on stack 'stack-bbl-env-huron-2017-07-19t09-00z'.
```

**Comment by evanfarrar** [comment]
Huh. I was able to complete `bbl up` on 3.2.6 in us-west-1. We use the API for asking which zones to install to, and have had other zones with non-sequential numbers before also, so it is a little strange this would happen. 

Barring an intermittent problem, there are two differences between my attempt to replicate your issue: my account sees us-west-1b and us-west-1c not a and c, and I did it on OSX. I may try a few other AWS accounts my team has as each one numbers zones differently to "load balance".
